### ISSUE TITLE
[Accessibility]  Add [lang] attribute to <html> element 

### ISSUE THREAD

**Issue by gracekishino** [gracekishino/coding_playlists#12]
Add [lang] attribute to `<html>` element in `index.html`.
 
 Like this: 
`<html lang="en">`

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
